Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:lts-slim AS frontend
+
+WORKDIR /app
+
+COPY package.json /app/
+RUN npm install
+COPY . /app/
+RUN npm run build
+
+FROM python:3.7-slim
+
+WORKDIR /app
+
+COPY api/requirements.txt /app/
+RUN pip install -U pip \
+    && pip install -r requirements.txt
```

Comment:
This could be further optimized by building the Python libs in a builder image and copied to the final image.

I kind of like the basket project's approach of using `python -m venv venv/`: https://github.com/mozmeao/basket/blob/master/Dockerfile